modules/apps/frontend-theme/frontend-theme-classic/src/css/_clay_variables.scss

```diff
@@ -10,7 +10,7 @@ $lead-font-weight: 400;
 $small-font-size: 0.875rem;
 
 $btn-outline-secondary: (
-	color: $dark-color,
+	color: #6b6c7e,
```

Comment:
I thought we were always using upper case for our HEX values... it doesn't look like that's the case in here which I'm not sure is a new issue or an old one...

Reply:
It looks like we have been using lower case for a long time and in other components like color picker are working on lowercase too so we can see in first place `#FFFFFF` but then if you modify it changes to `#ffffff`.
I going to move all uppercases to lower cases in `frontend-token-definition.json` :-D